### PR TITLE
Feature/airlock streaming

### PR DESCRIPTION
**Pull request by emm1R**
Enable streaming when uploading files with airlock by sending the checksum in the manifest file.

Other changes:
- public-key argument https://gitlab.ci.csc.fi/sds-dev/sd-desktop/sda-filesystem/-/issues/12
- Update segment container name https://gitlab.ci.csc.fi/sds-dev/sd-desktop/sda-filesystem/-/issues/8

### CODE REVIEW

File: cmd/airlock/main.go [hunk 1]
```diff
@@ -13,28 +13,44 @@ import (
...
+}
+
 func usage(selfPath string) {
 	fmt.Println("Password is read from environment variable CSC_PASSWORD")
 	fmt.Println("If this variable is empty airlock requests the password interactively")
 	fmt.Println("Usage:")
 	fmt.Println(" ", selfPath, "[-segment-size=sizeInMb] "+
 		"[-journal-number=journalNumber] [-original-file=unecryptedFilename] "+
-		"[-quiet] "+"username container filename")
+		"[-public-key=publicKeyFile] [-quiet] username container filename")
```
Comment (blankdots) [inline cmd/airlock/main.go:34]: we need to add this info to main readme

**Review by blankdots — CHANGES_REQUESTED** [review]
i think it might be cleaner if we don't place the public keys for test as files, we can use vars for test see https://github.com/neicnordic/crypt4gh/blob/master/keys/keys_test.go


missing changelog and version bump

**Comment by emm1R** [comment]
> i think it might be cleaner if we don't place the public keys for test as files, we can use vars for test see https://github.com/neicnordic/crypt4gh/blob/master/keys/keys_test.go
> 
> missing changelog and version bump

But then the tests won't work

**Comment by blankdots** [comment]
waiting to be tested

**Comment by teemukataja** [comment]
- [x] tested airlock proxy export works with multiple public keys
- [ ] manifest file to be tested, pending airlock api update

**Pull request closed without merging**